src/common/java/net/minecraftforge/gradle/common/util/Utils.java

```diff
@@ -491,4 +497,27 @@ public static String replaceTokens(Map<String, ?> tokens, String value) {
 
         return buf.toString();
     }
+
+    public static void setupIDEResourceCopy(@Nonnull final Project project) {
+        if (project.getPlugins().hasPlugin(IdeaPlugin.class)) {
+            final IdeaPlugin idea = project.getPlugins().getPlugin(IdeaPlugin.class);
+            project.getTasks().create("copyIdeaResources", CopyIDEAResources.class, task -> task.configure(idea.getModel()));
+        } else {
+            project.getLogger().warn("'idea' plugin is not found! IDEA resources will not be copied.");
+            project.getTasks().create("copyIdeaResources", Task.class); // Create dummy class
+        }
+
+        if (project.getPlugins().hasPlugin(EclipsePlugin.class)) {
+            project.getTasks().create("copyEclipseResources", CopyEclipseResources.class, task -> {
+                task.dependsOn("eclipse");
+                project.getTasks().named("eclipse").configure($ -> $.doLast($$ -> {
+                    final EclipseModel eclipse = project.getExtensions().findByType(EclipseModel.class);
+                    task.configure(eclipse, project);
+                }));
+            });
+        } else {
+            project.getLogger().warn("'eclipse' plugin is not found! Eclipse resources will not be copied.");
+            project.getTasks().create("copyEclipseResources", Task.class); // Create dummy class
+        }
```

Comment:
Perhaps we should enable both plugins automatically if this option is enabled?

Reply:
The presence of the eclipse plugin will make the `copyEclipseResources` task depend on the `eclipse` task for proper configuration. The `eclipse` task will generate the eclipse files.

Reply:
And only people in eclipse will be running the eclipse task... Also this seems like bad design, running `eclipse` is no longer necessary to import into eclipse, so this should be designed to be more versatile.